app/assets/stylesheets/common/_navigation.scss

```diff
@@ -452,7 +436,7 @@ li#dropdown {
     }
 
     ul {
-      @include border-radius(0);
+      border-radius: 0;
```

Comment:
Properties should be ordered background, border-left, border-radius, border-right, border-top, color, display, left, margin-top, min-width, position, text-indent, top, width